src/SchemaDesign/SchemaDesignerQuestion.tsx

```diff
@@ -0,0 +1,81 @@
+/* eslint-disable dot-notation */
```

Comment:
We don't need to disable this rule, I've commented on the line below with more info.